Gillian-C/scripts/testSymb.sh

```diff
@@ -8,33 +8,34 @@ if [[ -z "${GITHUB_ACTIONS}" ]]; then
   source exec.env
 fi
 
+# TODO (Alexis): Make incremental analysis thread-safe to allow the use of --parallel
 echo "--- testing SLL ---"
-time gillian-c wpst symbolic/sll.c -l disabled --parallel
+time gillian-c wpst symbolic/sll.c -l disabled
```

Comment:
I don't know if the --parallel should be removed in the testSymb, this file is not for incremental use ?